### PR TITLE
Redesign: Sidebar/Drawer, AppBar

### PR DESCRIPTION
**Pull request by observerjnr**
Applied new design to the Drawer section and AppBar.
Added minor style fixes as well.

### CODE REVIEW

File: Examples/src/components/Navigation/ListItemsBlock.module.scss [hunk 1]
```diff
@@ -25,11 +25,11 @@
...
 
     .ExampleLink {
-        @extend %gutter-bottom;
-
         font-size: 0.875rem;
-        line-height: 1.5;
+        line-height: 17px;
         margin-left: 40px;
+        margin-top: 9px;
+        margin-bottom: 10px;
```
Comment (danylboiko95) [inline Examples/src/components/Navigation/ListItemsBlock.module.scss:32]: change it to one line margin: .... please

File: Examples/src/components/Search/Search.tsx [hunk 2]
```diff
@@ -9,15 +9,17 @@ import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles(theme => ({
     root: {
-        width: "100%",
+        display: "none",
         [theme.breakpoints.up("sm")]: {
-            width: "80ch"
+            width: "100%",
```
Comment (danylboiko95) [inline Examples/src/components/Search/Search.tsx:14]: Could you remove it to scss file, please

**Review by danylboiko95 — CHANGES_REQUESTED** [review]

File: Examples/src/components/Navigation/ListItemsBlock.module.scss (hunk 1 above)
Comment (observerjnr) [reply Examples/src/components/Navigation/ListItemsBlock.module.scss]: Fixed

File: Examples/src/components/Search/Search.tsx (hunk 2 above)
Comment (observerjnr) [reply Examples/src/components/Search/Search.tsx]: Extracted all other material-ui styles for this component into a stylesheet

File: Examples/src/components/AppTopBar/AppBarTop.tsx [hunk 3]
```diff
@@ -15,12 +16,13 @@ type TProps = {
...
 const AppBarTop: React.FC<TProps> = props => {
     const { toggleDrawer } = props;
+    const history = useHistory();
 
 
     return (
         <AppBar position="sticky">
             <Toolbar className={classes.AppBar}>
-                <img className={classes.Logo} src={Logo} alt="scichart-logo" />
+                <img className={classes.Logo} src={Logo} alt="scichart-logo" onClick={() => history.push("/")} />
```
Comment (danylboiko95) [inline Examples/src/components/AppTopBar/AppBarTop.tsx:25]: create this as a link, and inside link img. This would be better for seo

**Review by danylboiko95 — CHANGES_REQUESTED** [review]

**Pull request merged**